CONTRIBUTION.md

```diff
@@ -26,4 +26,11 @@ Your pull requests are welcome; however, they may not be accepted for various re
 
 ## Cross-Links
 
-To properly link from one page in the user-manual docs to another, use the syntax `[link name](../page/)` to link to the top level, or `[link name](../page/#anchor)` to link to an anchor (like a heading) in another document.  Note that `page` does _not_ include the `.md` extension.  The anchor names are the lower-case version of the section headers, with spaces replaced by hyphens.  Thus, from the [Themes](content/docs/themes/) page, link to the [**Preferences > Style Configurator**](content/docs/preferences/#style-configurator) using the syntax `[**Preferences > Style Configurator**](../preferences/#style-configurator)`.
+:to link from one page in the user-manual docs to another, use the syntax `[link name](../page/)`
+:to link to the top level, or `[link name](../page/#anchor)` 
+:to link to an anchor (like a heading) in another document. 
+*Note* that `page` does _not_ include the `.md` extension.  
+The anchor names are the lower-case version of the section headers, with spaces replaced by hyphens.  
+: Thus, from the [Themes](content/docs/themes/) page,
+:: link to the [**Preferences > Style Configurator**](content/docs/preferences/#style-configurator)
+::: using the syntax `[**Preferences > Style Configurator**](../preferences/#style-configurator)`.

```

Comment:
I am not sure what rendering you were intending.  But having a whole bunch of colons inline with the paragraph probably isn't it.

![image](https://user-images.githubusercontent.com/17455758/121400651-b184e800-c90c-11eb-87ec-3891d2d3e7b1.png)